flytekit/core/map_task.py

```diff
@@ -66,20 +67,23 @@ def __init__(
         )
 
     def get_command(self, settings: SerializationSettings) -> List[str]:
-        return [
+        container_args = [
             "pyflyte-map-execute",
-            "--task-module",
-            self._run_task._task_function.__module__,
-            "--task-name",
-            f"{self._run_task._task_function.__name__}",
             "--inputs",
             "{{.input}}",
             "--output-prefix",
             "{{.outputPrefix}}",
             "--raw-output-data-prefix",
             "{{.rawOutputDataPrefix}}",
+            "--resolver",
```

Comment:
this implies that the resolver has to be the last arg only?

Reply:
why not use multi-valued args? 
--resolver-arg "x=y" --resolver-arg "..."

Reply:
not sure what you mean.  the resolver can be specified anywhere, but the resolver args have to be at the end.

Reply:
I think @kumare3 means an alternative syntax is
`--resolver ... --resolver-arg "task-module=<TASK_MODULE>" -- resolver-arg "task-name=<TASK_NAME>"`

Where `--resolver-arg` is a click option instead of an argument.

Will the resolver always have 2 args, `task-module` and `task-name`?

Reply:
just throwing out another idea:
```
--resolver ... --resolver-task-module ... --resolver task-name ...
```